core/shared/src/main/scala/com/monovore/decline/opts.scala

```diff
@@ -14,7 +14,7 @@ class Command[+A] private[decline](
 
   def showHelp: String = Help.fromCommand(this).toString
 
-  def parse(args: Seq[String]): Either[Help, A] = Parser(this)(args.toList)
+  def parse(args: Seq[String], env: Map[String, String]): Either[Help, A] = Parser(this)(args.toList, env)
```

Comment:
I'd be tempted to add a default value of `Map.empty` for this, to avoid breaking existing usages.